internal/checker/terraform/rules/gcp/google_project_iam_binding_user_token_creator_restricted.rego

```diff
@@ -0,0 +1,59 @@
+# METADATA
+# title: "Verify that Service Account User and Service Account Token Creator roles are not allocated to IAM users at the project level"
+# description: "Assure that IAM users are not designated with either the Service Account User or Service Account Token Creator roles at the project level to enhance security."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project_iam.html#google_project_iam_binding
+# custom:
+#   id: CB_TFGCP_003
+#   severity: HIGH
+package lib.terraform.CB_TFGCP_003
+
+import future.keywords.in
+
+isvalid(block) {
+	supported_resources := ["google_project_iam_binding", "google_project_iam_member"]
+	block.Type == "resource"
+	some label in block.Labels
+	label in supported_resources
+}
+
+fail[block] {
+	some block in input
+	isvalid(block)
+	block.Attributes.role == "roles/iam.serviceAccountTokenCreator"
+}
+
+fail[block] {
+	some block in input
+	isvalid(block)
+	block.Attributes.role == "roles/iam.serviceAccountUser"
+}
+
```

Comment:
move the logic into pass like the script below

pass[block] {
	some block in input
	isvalid(block)
        invalid_roles := ["roles/iam.serviceAccountTokenCreator", "roles/iam.serviceAccountUser"]
	not block.Attributes.role in invalid_roles
}